common/history/states/dvg_ameriga.txt

```diff
@@ -156,40 +156,40 @@
 		
 		add_claim = c:TWT
 	}
-	s:STATE_CUZCO = {
```

Comment:
Not sure if you changed states. But this will break save game if you did. Which we can't do in minor patches.

Reply:
Senka said this should only change owned states at game start